### DISCUSSION TITLE
my GitHub support ticket is taking too long to get resolved #3097664

### DISCUSSION THREAD

**Discussion by omar-m-z-git** [community/community#144776] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

It has been 24 hours already since I last sent the support ticket [#3097664](https://support.github.com/ticket/3097664) even though the solution seems to be on the support's end and is very simple. Just remove the flag on my free GitHub account "omar-m-z-dotcom". Currently I haven't received any replies from support to resolve or even explain to me why the GitHub algorithm chose to flag the account "omar-m-z-dotcom". What is the holdup? Why is it taking so long to get any response?

**Comment by MatinGhanbari** [comment]
Don't worry these are the christmas days so that is why everything is blocked XD
I hope they will reply your ticket soon, best of luck :)

**Comment by AmineRACHID** [comment]
+1

**Comment by OmarrSakr** [comment]
Hi @omar-m-z-git,

I understand your frustration with the delay in resolving your GitHub support ticket. It’s always frustrating when an issue seems simple, yet takes longer than expected to resolve.

There could be several reasons for the delay:

Support Ticket Volume: GitHub support may be experiencing a high volume of tickets, especially around peak times, which can cause delays in responses.
Account Review Process: Sometimes, account flags require further investigation or review from multiple teams, which can take time.
Automated Systems: If the issue is related to a flag triggered by an automated system, it may require additional manual review or confirmation from the relevant team to ensure everything is in compliance.
Here are a couple of steps you can take:

Follow up: If it’s been over 24 hours, I recommend following up with another polite request for an update. Reference your ticket number and reiterate the urgency.
GitHub Community: If the issue is not progressing, consider reaching out through GitHub's official community forums. Sometimes other users or GitHub team members can provide insights or escalate issues more quickly.
You can also check if there are any platform-wide issues or announcements that might be delaying support.

Hopefully, you’ll get the resolution soon! If I can assist further, let me know.

And if this helps, don’t forget to mark it as the answer. 😊
